scripts/zones/Bostaunieux_Oubliette/mobs/Shii.lua

```diff
@@ -16,6 +16,18 @@ entity.onAdditionalEffect = function(mob, target, damage)
     return xi.mob.onAddEffect(mob, target, damage, xi.mob.ae.TERROR)
 end
 
+entity.onMobRoam = function(mob)
+    if VanadielHour() >= 6 and VanadielHour() < 18 then -- Despawn if its day
+        DespawnMob(mob:getID())
+    end
+end
+
+entity.onMobDisengage = function(mob)
+    if VanadielHour() >= 6 and VanadielHour() < 18 then -- Despawn if its day
```

Comment:
Don't think this is needed or exactly accurate. when the mob goes idle, it will trigger the code in onMoBRoam and do this. This also provides the very tiny window to reclaim in before the depop like retail does.

Reply:
Removed the onMobDespawn from Manes and Shii